spring-boot-dependencies/pom.xml

```diff
@@ -1295,6 +1295,11 @@
 				<version>${tomcat.version}</version>
 			</dependency>
 			<dependency>
+				<groupId>org.apache.tomcat</groupId>
+				<artifactId>tomcat-catalina-jmx-remote</artifactId>
```

Comment:
Dependencies in this file are ordered. Could you please move it at the right place?